src/utils/functions.ts

```diff
@@ -0,0 +1,7 @@
+export const getAccessToken = (): string | null => {
```

Comment:
create a file localstorage and move all local storage relatad function into that, better to create a hook to use localstorage or you can use library `'use-hook-ts'`